ibm/service/project/data_source_ibm_project_config.go

```diff
@@ -337,11 +336,84 @@ func DataSourceIbmProjectConfig() *schema.Resource {
 				Computed:    true,
 				Description: "The flag that indicates whether a configuration update is available.",
 			},
+			"template_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The stack definition identifier.",
+			},
+			"member_of": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Optional:    true,
```

Comment:
This datasource can we remove the Optional true I think its computed only

Reply:
fixed